src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import duke.dependencies.parser.Controller;
+
+
+/**
+ * Class that separates the command checker/parser from the main(). Catches command that
+ * ends/closes program, terminating it immediately. Prints to std::out.
+ *
+ */
+class Ui {
+    private static final String DIVIDER = "____________________________________________________________\n";
+    private static final String GREETING = "Hello, I'm Duke\nwhat can I do for you?\n";
+    private static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String CIAO = DIVIDER + "Spero di rivederti presto\n" + DIVIDER;
+    private static final String CONVO_START = DIVIDER + LOGO + "\n" + GREETING + DIVIDER;
+    private static final String END = "end|ciao|bye|close|exit|nights|shutdown";
```

Comment:
Since these constants are associated, it might make sense to prefix them with a common name (eg. MSG_DIVIDER, MSG_GREETING, ...).